core/state/state_object.go

```diff
@@ -800,7 +800,11 @@ func (s *stateObject) fetchExpiredFromRemote(prefixKey []byte, key common.Hash,
 
 	// if no prefix, query from revive trie, got the newest expired info
 	if resolvePath {
-		_, err := tr.GetStorage(s.address, key.Bytes())
+		val, err := tr.GetStorage(s.address, key.Bytes())
```

Comment:
pls add a TODO to alert the potential problem, like `temporary fix snap expired, but trie not expire, may investigate more later.`